README.md

```diff
@@ -32,13 +32,15 @@ Start by reaching out on the community channels (Reddit, Discord, IRC, etc.
 see the [Community Channels](http://docs.godotengine.org/en/stable/community/channels.html) doc),
 then create your proposal once you have gained some interest.
 
-5. You can make a PR implementing the feature in the main repository before 
+5. If a post receives little or no engagement after 30 days it will be closed.
```

Comment:
```suggestion
5. If a post receives little or no engagement after 30 days, it will be closed.
```

Also, we might want to reword it to be able to close issues that *are* actively commented, but mostly receive negative feedback.

Reply:
I think there should be a separate rule for negative feedback. That way we can be very clear about which rule relates to closing the issue.